frame/uniques/src/weights.rs

```diff
@@ -77,19 +76,19 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Uniques Class (r:1 w:1)
 	// Storage: Uniques ClassAccount (r:0 w:1)
 	fn create() -> Weight {
-		(24_063_000 as Weight)
+		(23_737_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	// Storage: Uniques Class (r:1 w:1)
 	// Storage: Uniques ClassAccount (r:0 w:1)
 	fn force_create() -> Weight {
-		(13_017_000 as Weight)
+		(13_334_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	// Storage: Uniques Class (r:1 w:1)
-	// Storage: Uniques Asset (r:1 w:0)
+	// Storage: Uniques Instance (r:1 w:0)
```

Comment:
You can see the mistake here too. None of these storage names should change to maintain backwards compatibility.